src/groups/mqb/mqbs/mqbs_filestoreprotocol.h

```diff
@@ -2107,38 +2107,39 @@ inline FileHeader& FileHeader::setMagic2(unsigned int value)
 
 inline FileHeader& FileHeader::setProtocolVersion(unsigned char value)
 {
-    d_protoVerAndHeaderWords = (d_protoVerAndHeaderWords &
-                                k_HEADER_WORDS_MASK) |
-                               (value << k_PROTOCOL_VERSION_START_IDX);
+    d_protoVerAndHeaderWords = static_cast<unsigned char>(
+        (d_protoVerAndHeaderWords & k_HEADER_WORDS_MASK) |
+        (value << k_PROTOCOL_VERSION_START_IDX));
     return *this;
 }
 
 inline FileHeader& FileHeader::setHeaderWords(unsigned char value)
 {
-    d_protoVerAndHeaderWords = (d_protoVerAndHeaderWords &
-                                k_PROTOCOL_VERSION_MASK) |
-                               (value & k_HEADER_WORDS_MASK);
+    d_protoVerAndHeaderWords = static_cast<unsigned char>(
+        (d_protoVerAndHeaderWords & k_PROTOCOL_VERSION_MASK) |
+        (value & k_HEADER_WORDS_MASK));
     return *this;
 }
 
 inline FileHeader& FileHeader::setBitness(Bitness::Enum value)
 {
     if (Bitness::e_64 == value) {
-        d_bitnessAndFileType = (d_bitnessAndFileType & k_FILE_TYPE_MASK) |
-                               (1 << k_BITNESS_START_IDX);
+        d_bitnessAndFileType = static_cast<char>(
+            (d_bitnessAndFileType & k_FILE_TYPE_MASK) |
+            (1 << k_BITNESS_START_IDX));
     }
     else {
-        d_bitnessAndFileType = d_bitnessAndFileType & k_FILE_TYPE_MASK;
+        d_bitnessAndFileType = d_bitnessAndFileType &
+                               static_cast<char>(k_FILE_TYPE_MASK);
```

Comment:
Can we wrap the entire expression to make the code more uniform?
```suggestion
        d_bitnessAndFileType = static_cast<char>(d_bitnessAndFileType &
                               k_FILE_TYPE_MASK);
```